app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1390,6 +1390,31 @@ body.dashboard-faq dl dt {
   border-bottom: none;
 }
 
+a:focus {
+  outline: 5px auto $codebar-blue;
+  color: darken($codebar-blue, 30%);
+}
+
+a.button:focus {
+  background: darken($codebar-blue, 50%);
+  border-color: darken($codebar-blue, 50%);
+  color: $white;
+  outline: none;
+}
+
+.top-bar-section li:not(.has-form) a:not(.button):focus,
+.title > a:focus {
+  background: darken($codebar-blue, 50%);
+  color: $white;
+  outline: none;
+}
+
+.main-footer a:focus {
```

Comment:
Same deal but in `assets/stylesheets/partials/_layout.scss` ?

```scss
 a {
    color: $white;
    border: none;

    &:focus {
        // awesome code
    }

    &:hover {
      color: $secondary-color;
    }
  }

```

There may be places for the other code? But I'll shut up until you've had a look. _If_ I am right all the css should probably find a new home.